### ISSUE TITLE
Different domains on analytic account in the expense form

### ISSUE THREAD

**Issue by max3903** [odoo/odoo#4828]
# Version

Version 7, but maybe on later one as well.
# Description

The domain on analytic account is different between the form view and the tree view for the expense line:

```
<field domain="[('type','=','normal')]" name="analytic_account" groups="analytic.group_analytic_accounting"/>
```

https://github.com/odoo/odoo/blob/7.0/addons/hr_expense/hr_expense_view.xml#L96

```
<field domain="[('type','in',['normal','contract'])]" name="analytic_account" groups="analytic.group_analytic_accounting"/>
```

https://github.com/odoo/odoo/blob/7.0/addons/hr_expense/hr_expense_view.xml#L115
# Current result

We can't select a project or contract in the form view
# Expected result

I can select a project or contract or an analytic account in the form view.

**Comment by llacroix** [comment]
From what I can see, the problem is present on both odoo 7 and 8. The patch can be applied to both version without any change. I can create a PR for version 7 and 8.

**Comment by Yenthe666** [comment]
Hi @max3903,

Thank you for your bug report and our apoligies that we haven't had the time to look at this bug report. We do our best to handle all of them, but we sometimes miss some.
At this point Odoo 8 is no longer supported as Odoo only supports three versions (9.0, 10.0, 11.0 at the moment).
It has been fixed in 9.0 and up so there is nothing to do anymore.

Regards,
Yenthe
